siphon/catalog.py

```diff
@@ -454,8 +454,10 @@ def __init__(self, element_node, catalog_url=''):
 
         """
         self.name = element_node.attrib['name']
+
         if 'urlPath' in element_node.attrib:
             self.url_path = element_node.attrib['urlPath']
+            self.id = element_node.attrib.get('ID', None)
```

Comment:
Did you really mean to move this line *into* the block? What I meant was that you could do, outside any if block
```python
self.id = element_node.attrib.get('ID', None)
self.url_path = element_node.attrib.get('urlPath', None)
```